mosu-app/src/pages/auth/member/signin.tsx

```diff
@@ -12,9 +12,18 @@ import { AuthSupportMenu } from "@/widgets/auth/AuthSupportMenu";
 import { InputFieldSet } from "@/shared/components/Input";
 import { Button } from "@/shared/ui/button";
 
+const KAKAO_CLIENT_ID = process.env.NEXT_PUBLIC_KAKAO_REST_API_KEY!;
+const REDIRECT_URI = process.env.NEXT_PUBLIC_KAKAO_REDIRECT_URI!;
+
+const kakaoAuthUrl = `https://kauth.kakao.com/oauth/authorize?client_id=${KAKAO_CLIENT_ID}&redirect_uri=${REDIRECT_URI}&response_type=code`;
+
```

Comment:
가능하면 경로는 상수처리하는게 관리하기 편할거같아요~
```suggestion
const kakaoAuthUrl = `NEXT_PUBLIC_KAKAO_REDIRECT_URI?client_id=${KAKAO_CLIENT_ID}&redirect_uri=${REDIRECT_URI}&response_type=code`;
```